website/source/docs/provisioners/ansible.html.md.erb

```diff
@@ -180,6 +180,13 @@ Optional Parameters:
 -   `user` (string) - The `ansible_user` to use. Defaults to the user running
     packer.
 
+-   `use_proxy` (boolean) - Whether or not to set up a localhost proxy adapter
+    so that Ansible has an IP address to connect to, even if your guest does not
+    have an IP address. For example, the adapter is necessary for Docker builds
+    to use the Ansible provisioner. Defaults to "true". If you set this option
```

Comment:
NIT: One thing we could do here is make docs generated from the `Config` struct 🤔

Reply:
I felt like that was outside of the scope of this PR; we can change to generating the docs in a separate, dedicated PR.

Reply:
Got it !